sites/gent/macros.tex

```diff
@@ -50,13 +50,12 @@
 \macro{\computenode}{node2001.delcatty.gent.vsc}
 \macro{\computenodeshort}{node2001}
 \macro{\defaultcluster}{delcatty}
-\macro{\othercluster}{raichu}
+\macro{\othercluster}{swalot}
 % Support
 \macro{\hpcinfo}{hpc@ugent.be}
 \macro{\hpcusersml}{hpc-users@lists.ugent.be}
 \macro{\hpcannounceml}{hpc-announce@lists.ugent.be}
-\macro{\operatingsystem}{CentOS 7.2 (phanpy, golett, swalot), Scientific Linux 6.7 (raichu, delcatty)}
-\macro{\operatingsystemSL}{Scientific Linux}
+\macro{\operatingsystem}{CentOS 7.5 (phanpy, golett, swalot, delcatty, victini, skitty)}
```

Comment:
Please list the cluster alphabetically (easier to check what's missing in the future)